src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -424,6 +424,13 @@ public enum ResizeDirection
         /// <since_tizen> 3 </since_tizen>
         public static Window Instance { get; internal set; }
 
+
+        /// <summary>
+        /// Gets the default window.
```

Comment:
you might consider adding sth more, like "Gets the default window - the initial window of the application.", maybe terms like "main window", "top window" do not fit or maybe the do. I just would prefer to have some explanation added about how it is default, which one is it, does it change, it is always available? On the other hand most users will find that they should just use this "Window.Default" and that will be all they need to know.

also, the extra empty line 427 is not needed

Reply:
@wiertel 
Thank you for reviewing. I have changed it according to your comments.